creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -3,14 +3,62 @@ const { rehydrateIpfsFromFsIfNecessary, rehydrateIpfsDirFromFsIfNecessary } = re
 const { logger: genericLogger } = require('./logging')
 const config = require('./config')
 const enableRehydrate = config.get('enableRehydrate')
+const redisClient = require('./redis')
 
 const PROCESS_NAMES = Object.freeze({
   rehydrate_dir: 'rehydrate_dir',
   rehydrate_file: 'rehydrate_file'
 })
 
+// minimum threshold per day to rehydrate
+const MIN_REHYDRATE_THRESHOLD = config.get('minRehydrateThreshold') || 10
```

Comment:
the ` || 10` is probably not necessary since the config defaults with 10?

Reply:
also, what was the decision behind the minimum threshold number of 10?

Reply:
i looked through the prod traffic and tried to pick a value higher than most CIDs which get requested